### PR TITLE
Update CLI compute resource tests

### PR DESCRIPTION
**Pull request by elyezer**
Merge similar update tests into one. In the merge was dropped an invalid
data (provider update is not allowed).

Fix test_create_positive_libvirt data to avoid bash complaining about
the generated command. Also update data entries string interpolation to
string formatting.

Test results:

```
$ py.test tests/foreman/cli/test_computeresource.py -k "test_update_positive or test_create_positive_libvirt"
================================ test session starts ================================
platform darwin -- Python 2.7.6 -- py-1.4.26 -- pytest-2.6.4
collected 29 items

tests/foreman/cli/test_computeresource.py ............

== 17 tests deselected by '-ktest_update_positive or test_create_positive_libvirt' ==
==================== 12 passed, 17 deselected in 156.60 seconds =====================
```

### CODE REVIEW

**Comment by elyezer** [comment]
Update: rebased

File: tests/foreman/cli/test_computeresource.py [hunk 1]
```diff
@@ -169,14 +134,14 @@ def test_delete(self):
...
-                  gen_string(str_type='alpha'),
-         u'description': '<html>%s</html>' %
-                         gen_string(str_type='alpha')},
-        {u'name': "%s[]@#$%%^&*(),./?\"{}><|''" %
-                  gen_string(str_type='utf8'),
-         u'description': "%s[]@#$%%^&*(),./?\"{}><|''" %
-                         gen_string(str_type='alpha')},
+        {u'name': u'<html>{0}</html>'.format(
+            gen_string(str_type='alpha')),
+         u'description': '<html>{0}</html>'.format(
```
Comment (Ichimonji10) [inline tests/foreman/cli/test_computeresource.py]: Do you want to add the `u` prefix to the second string on this line?

**Comment by Ichimonji10** [comment]
ACK pending comment

File: tests/foreman/cli/test_computeresource.py (hunk 1 above)
Comment (elyezer) [inline tests/foreman/cli/test_computeresource.py]: Sure, thank you

**Comment by elyezer** [comment]
Update: comment addressed

File: tests/foreman/cli/test_computeresource.py [hunk 2]
```diff
@@ -169,14 +134,14 @@ def test_delete(self):
...
-         u'description': "%s[]@#$%%^&*(),./?\"{}><|''" %
-                         gen_string(str_type='alpha')},
+        {u'name': u'<html>{0}</html>'.format(
+            gen_string(str_type='alpha')),
+         u'description': u'<html>{0}</html>'.format(
+             gen_string(str_type='alpha'))},
+        {u'name': u"{0}[]@#$%^&*(),./?\\\"{{}}><|''".format(
+            gen_string(str_type='utf8')),
+         u'description': u"{0}[]@#$%^&*(),./?\\\"{{}}><|''".format(
+             gen_string(str_type='alpha'))},
```
Comment (JacobCallahan) [inline tests/foreman/cli/test_computeresource.py]: nice formatting change. not a blocker, but any reason for keeping the str_type= in the argument?

Comment (elyezer) [inline tests/foreman/cli/test_computeresource.py]: @JacobCallahan sharp eyes. Will drop that as we does not use in new code.

**Comment by elyezer** [comment]
Update: comment addressed and rebased

**Comment by JacobCallahan** [comment]
Perfect! ACK

**Pull request merged**